app/views/games/index.html.erb

```diff
@@ -1 +1,165 @@
-<h1>StoicChess</h1>
\ No newline at end of file
+<br />
+<section id="register">
+  <div class="col-sm-5 col-sm-offset-1">
+    <%= image_tag 'chess.jpg', class: 'img-responsive feature_pic' %>
+    <br />
+  </div>
+  <div class="col-sm-3 col-sm-offset-1">
+    <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+        <div class="form-inputs">
+          <br />
+          <br />
+          <br />
+          <%= f.input :email, required: false, autofocus: true %>
+          <br />
+          <%= f.input :password, required: false %>
+          <br />
+          </div>
+          <%= f.input :remember_me, as: :boolean if devise_mapping.rememberable? %>
+          <%= render "devise/shared/forgotpassword" %>
+          <br />
+        <div class="form-inputs">
+          <%= f.button :submit, "LOG IN", class: 'btn btn-success login' %>
+        </div>
+    <% end %>
+  </div>
+</section>
+<br class="clr" />
+<hr class="col-sm-offset-1 col-sm-10"/>
+<br />
+<section id="features">
+  <div class="col-sm-offset-2 col-sm-8 text-center">
+    <h2>CHESS APP - FINAL PROJECT</h2>
+    <h4>Description of the project and fact that it is our final project for FHP</h4>
+    <br />
+  </div>
+  <div class="col-sm-offset-2 col-sm-2 col-xs-offset-2 col-xs-8 text-center"/>
+    <%= image_tag 'chess.jpg', class: 'img-responsive feature_pic' %>
+    <h3>Feature 1</h3>
+    <p>Words words words, words words. Words words words, words words. Words words words, words words. Words words words, words words.</p>
+  </div>
+  <div class="col-sm-offset-1 col-sm-2 col-xs-offset-2 col-xs-8 text-center"/>
+    <%= image_tag 'chess.jpg', class: 'img-responsive feature_pic' %>
+    <h3>Feature 2</h3>
+    <p>Words words words, words words. Words words words, words words. Words words words, words words. Words words words, words words.</p>
+  </div>
+  <div class="col-sm-offset-1 col-sm-2 col-xs-offset-2 col-xs-8 text-center"/>
+    <%= image_tag 'chess.jpg', class: 'img-responsive feature_pic' %>
+    <h3>Feature 3</h3>
+    <p>Words words words, words words. Words words words, words words. Words words words, words words. Words words words, words words.</p>
+  </div>
+  <br class="clr" />
+</section>
+<hr class="col-sm-offset-1 col-sm-10"/>
+<section id="team">
+  <div class="text-center">
+    <h2>THE TEAM</h2>
+    <br class="clr" />
+  </div>
+  <div class="col-sm-offset-1 col-sm-2"/>
+    <%= image_tag 'chad.png', class: 'img-responsive team_profile' %>
+    <h5 class="text-center">Chad</h5>
+    <%= image_tag 'linked_in.png', class: 'link_icon' %>
```

Comment:
Bootstrap has built in http://glyphicons.com/ - I recommend using those for the social icons.